### PR TITLE
Support shared drive

### PR DESCRIPTION
**Pull request by loadingthedev**
Pr for the issue #3

### CODE REVIEW

**Comment by loadingthedev** [comment]
wait i forgot to check search functionality.
don't merge now . 
i will check that also then you can merge after that.

File: src/config/gIndex.config.ts [hunk 1]
```diff
@@ -39,8 +39,8 @@ const config: gIndexConfig = {
      * You need to create a new folder and share it with the service account
      * Then, copy the folder id and paste it here
      */
-    rootFolder: "1KgPV6QB1GYT8fmn2uTfbtr9rDXqcRR0j",
-
+    rootFolder: "",
```
Comment (mbahArip) [inline src/config/gIndex.config.ts:42]: Can you revert the root folder id to the previous value so it doesn't break the current demo site?
After that i will approve the PR

**Comment by mbahArip** [comment]
Ok

File: src/config/gIndex.config.ts (hunk 1 above)
Comment (loadingthedev) [reply src/config/gIndex.config.ts:42]: ok sure

**Comment by loadingthedev** [comment]
@mbahArip done now you can check and merge the pr

**Pull request merged**